x-pack/solutions/security/plugins/security_solution/public/helpers.tsx

```diff
@@ -234,9 +234,16 @@ export const isSubPluginAvailable = (pluginKey: string, capabilities: Capabiliti
   if (CASES_SUB_PLUGIN_KEY === pluginKey) {
     return capabilities[CASES_FEATURE_ID].read_cases === true;
   }
-  return capabilities[SERVER_APP_ID].show === true;
+  return hasAccessToSecuritySolution(capabilities);
 };
 
+export function hasAccessToSecuritySolution(capabilities: Capabilities) {
```

Comment:
Nit: is "having access" synonymous with having the `show` capability? Is there a more accurate phrasing we could use here?

Reply:
Yep, `show` grants you access to security solution